setup.cfg

```diff
@@ -14,7 +14,6 @@ norecursedirs =
     .env
 testpaths =
     coalib
-    docs
```

Comment:
I'm against this: https://github.com/coala-analyzer/coala/issues/1790#issuecomment-219034544